app/models/reporting/hud_reports/report_instance.rb

```diff
@@ -0,0 +1,14 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module Reporting::HudReports
+  class ReportInstance < ReportingBase
+    self.table_name = 'hud_report_instances'
+
+    belongs_to :user
```

Comment:
I wonder if we should add a belongs to CoC as well.  HUD reports should never be run for more than one CoC at a time and that might make limiting access easier in the future.  It could connect up to the Consent CoC table.